spikeinterface/widgets/ipywidgets/utils.py

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
For this module I think I would done classes get value retriever.
But lets go with this approach for now.